scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -1633,7 +1665,7 @@ function main {
           println DEBUG "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
           println OFF " Pool Management\n"\
 						" ) New      - create a new pool"\
-                                                " ) Import   - import node cold keys from hardware"\
+						" ) Import   - import node cold keys from Ledger HW device (Trezor not supported)"\
```

Comment:
Not sure if we should commit trezor not supported string completely, as it's a state that can change again in future with minimal/no change to code

Reply:
Pool keys on Ledger was added a long time ago. Over a year I think. I haven't heard of it being worked on for trezor so my guess is it wont be added any time soon, if ever. 

If we dont include any text about it users might think trezor should be supported.

Reply:
maybe can use as below , specifying what we can test - and leave the rest to upstream?
```suggestion
						" ) Import   - import node cold keys from HW device (only Ledger has been tested)"\
```

Reply:
Though I understand your intention I want to leave text to make it clear that only Ledger is supported. But I rephrased it a bit. If Trezor should decide to support this down the road, its a small change to update text. Rephrasing it as only Ledger tested still suggests that Trezor could work while it's not supported on the device.